bitnami/postgresql/Chart.yaml

```
@@ -35,4 +35,4 @@ maintainers:
 name: postgresql
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/postgresql
-version: 16.4.16
+version: 16.4.17
```

Since this adds a new feature to the chart, we should bump a minor version:

```suggestion
version: 16.5.0
```